src/components/footer/Footer.jsx

```diff
@@ -59,9 +109,45 @@ export const Footer = () => {
                     More about CORA’L
                 </AccordionSummary>
             }
-            <AccordionDetails sx={{ background: "#1B4B66", color: "#fff" }}>
-                <FooterList title={"Shop by Category"} footerLinks={links} />
-                <FooterList isMobile={isMobile} type={"row"} title={"Shop by Category"} footerLinks={links} />
+
+            <AccordionDetails sx={{
+                background: "#1B4B66", color: "#fff", padding: conditionalStyles.padding, display: "flex", flexWrap: "wrap", justifyContent: "space-between"
+            }}>
+
+                <Box sx={{
+                    display: "flex", gap: conditionalStyles.gap, flexWrap: "wrap", width: conditionalStyles.width,
+                    paddingBlock: conditionalStyles.paddingBlock, paddingInline: conditionalStyles.paddingInline
+                }}>
+                    <FooterList title={"Shop by Category"} footerLinks={links} />
```

Comment:
When you have a context already don't add extra context the prop name can just be links no need to add footer to it since it's on a component named FooterList